openssl/src/x509/mod.rs

```diff
@@ -1421,6 +1422,37 @@ impl X509ReqRef {
     }
 }
 
+/// The reason that a certificate was revoked.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct CrlReason(i64);
```

Comment:
This feels messy but the ASN1_ENUMERATED `get_int64` methods forces my hand slightly.

Reply:
I've changed this to use `c_int` as it's now the caller's problem.